src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -83,6 +85,18 @@ export default class MeetingContainer extends Component {
     }
   }
 
+  componentWillUnmount() {
+    // Time for clean up!
+    this.resetContainer();
+  }
+
+  resetContainer() {
+    const {dispatch, teamSub, memberSub} = this.props;
+    teamSub.unsubscribe();
```

Comment:
we shouldn't have to unsubscribe from either. even if a previous subscribe were cached, the cashay subscription cache would keep them separated based on variables.